starter-code/css/main.css

```diff
@@ -0,0 +1,96 @@
+h1 a {
```

Comment:
It's good practice to keep the order of your selectors in your CSS file from least specific at the top to most specific at the bottom. SO you should start by applying styles to the `<body>` at the top of this file and trickle down with more specific selectors as you go.